pulsar-broker/src/test/resources/authentication/basic/.htpasswd

```diff
@@ -0,0 +1,2 @@
+superUser:mQQQIsyvvKRtU
```

Comment:
I am not sure but should we add this file under [license exclude list](https://github.com/apache/incubator-pulsar/blob/master/pom.xml#L609) ?

Reply:
I added it.